steps/swift-environment-variable-injector/0.1.0/step.yml

```diff
@@ -0,0 +1,70 @@
+title: Swift Environment Variable Injector
+summary: |
+  Step to inject static values from environment variables into Swift code files before compilation and Continuous deployment allowing us to not expose our production keys and values in statically in our code and store sensitive values  safe.
+description: |
+  TThis step allows us to inject CI pipelines environment variables values into Swift code  static values before compilation and Continuous deployment to a specific environment(Development, Testing or Production) where we can define the values injected for each build and deployment e.g. a API url that the App uses and is different for each environment. Also it allows us to not expose our production keys and values in statically in our code.
```

Comment:
you can use markdown format, to make the description more readable, if you want to.